### ISSUE TITLE
error: couldn't read version from server (installing kubernetes on CoreOS)

### ISSUE THREAD

**Issue by mmobini** [kubernetes/kubernetes#10539]
I've install and configure kuberenetes on top of CoreOS Cluster based on [this article](https://www.digitalocean.com/community/tutorials/how-to-install-and-configure-kubernetes-on-top-of-a-coreos-cluster).

I see in similar  issues that maybe the problem is dependency of kubelet service on docker but after removing dependency same error occured when i run different commends like this:

```
./kubectl get pods

error: couldn't read version from server: Get http://localhost:8080/api: dial tcp 127.0.0.1:8080: connection refused
```

Could Anybody help me overcome this issue?

**Comment by zmerlynn** [comment]
That article is quite old. Were you still trying to configure Kubernetes 0.7.0? If not, a *ton* of things have changed. You might try comparing that article to one of the current getting started guides, which you can find here: https://github.com/GoogleCloudPlatform/kubernetes/tree/master/docs/getting-started-guides.

**Comment by suyogbarve** [comment]
Seems like something might be broken with latest release
yum - install kubernetes (results in the above problem.)

However the problem was resolved on installing *0.20.0-0.3.git835eded.fc21.x86_64.rpm

**Comment by rupsee** [comment]
I follwed the Getting Started guide from http://kubernetes.io/v1.0/docs/getting-started-guides/vagrant.html
and tried deploying a vagrant based cluster, both by building the source code and then when that didn't work, using the curl script. In both cases the cluster seems to be set up and I can do a vagrant ssh to the master and a minion, but running kubectl get pods or for that matter almost all kubectl commands return:

error: couldn't read version from server: Get http://localhost:8080/api: dial tcp 127.0.0.1:8080: connection refused

I read somewhere that this could happen if your config was faulty, so I tried kubectl.sh config info and this is what I get:
./cluster/kubectl.sh config view
apiVersion: v1
clusters: []
contexts: []
current-context: ""
kind: Config
preferences: {}
users: []

I'd gotten this to work in a different machine successfully with v1beta3. Unfortunately I don't have access to that machine any more so don't know what the configuration had been set to.

Any help would be much appreciated.

Cheers
--Rupa
